subprojects/logging/src/integTest/groovy/org/gradle/internal/logging/console/jvm/AbstractConsoleJvmTestWorkerFunctionalTest.groovy

```diff
@@ -99,11 +99,11 @@ class ConsoleJvmTestWorkerFunctionalTest extends AbstractConsoleFunctionalSpec {
         JavaTestClass.SHORTENED_TEST1 | JavaTestClass.SHORTENED_TEST2 | 'shortened'
     }
 
-    private String junitTest(String testClassName, String serverResource) {
+    private String testClass(String testClassName, String serverResource) {
         """
             package org.gradle;
 
-            import org.junit.Test;
+            import ${testAnnotationClass()};
 
             public class $testClassName {
                 @Test
```

Comment:
:x: This probably works by chance, because both Junit and TestNG use `@Test`. Should be `${testAnnotationClass()}`

Reply:
I don't quite understand the comment. Can you elaborate? The import of the test class is a different one per sub class (either `ConsoleJUnitTestWorkerFunctionalTest` or `ConsoleTestNGTestWorkerFunctionalTest`). Sure, I could use the fully-qualified class name right here but there should be no difference.

Reply:
I mean that if the test class annotation happened to be, for a 3rd framework we support, `@Spec`, then the test would fail. No big deal.